03week/towersOfHanoi.js

```diff
@@ -19,24 +19,45 @@ function printStacks() {
   console.log("c: " + stacks.c);
 }
 
-function movePiece() {
-  // Your code here
+function movePiece(startStack,endStack) {
```

Comment:
missing additional tests
missing readme code plan file.

Reply:
Hey Yousif! My README & additional tests stayed in the old file where I attempted gui: https://github.com/r-pm/javascript-workbook/tree/rpm/checkpoint1/checkpoint1